whistl/microarray_rnaseq_gene_intersection.py

```diff
@@ -16,11 +16,13 @@
 # List everything in data_dir
 subfiles = [os.path.join(train_dir, f) for f in os.listdir(train_dir)]
 # Keep only data directories, not anything else that might be in data_dir
-train_dirs = [f for f in subfiles if ('SRP' in f or 'GSE' in f) and os.path.isdir(f)]
+train_dirs = [f for f in subfiles if ('SRP' in f or 'GSE' in f or 'E-M' in f)
```

Comment:
That's a good idea. Since I'm switching to compendium data, I won't have to calculate the intersection, so it may be overkill to make a config file currently. If I end up switching back to data directories or build a config file in the future, I'll add this in